interfaces/builtin/network_manager.go

```diff
@@ -296,6 +296,14 @@ dbus (receive, send)
     path=/org/freedesktop
     interface=org.freedesktop.DBus.ObjectManager
     peer=(label=###SLOT_SECURITY_TAGS###),
+
+# nmcli uses this in newer versions
+dbus (receive, send)
```

Comment:
Does this need `receive` or is `send` sufficient?

Reply:
Good point, I tried and `send` seemed to be enough - I've removed the `receive` now.